internal/db/schema/migrations/postgres/2/05_job.up.sql

```diff
@@ -0,0 +1,79 @@
+begin;
+
+create table job (
+    id text primary key,
+    name wt_name not null,
+    description wt_description not null,
+    code text not null
+        constraint job_code_must_be_not_empty
+            check(length(trim(code)) > 0),
+    next_scheduled_run timestamp with time zone not null
+        default current_timestamp,
+
+    constraint job_name_code_uq
+        unique(name, code)
+);
+
+comment on table job is
+    'job is a base table where each row represents a unique job that can only have one running instance at any specific time.';
+
+create table job_name_enm (
```

Comment:
None of the other tables reference to this table. Are any of the tables missing foreign keys to this?